homeassistant/components/citybikes/strings.json

```
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Pick CityBike network",
+        "data": {
+          "network": "Network",
+          "station_filter": "Which stations to monitor",
+          "name": "Hub Name"
```

HA requires sentence-casing for all strings:

```suggestion
          "name": "Hub name"
```